server/config/cloudinaryconfig.js

```diff
@@ -0,0 +1,49 @@
+import multer from 'multer';
+import cloudinary from 'cloudinary';
+import cloudinaryStorage from 'multer-storage-cloudinary';
+import dotenv from 'dotenv';
+import Response from '../utils/response.util';
+
+
+dotenv.config();
+
+cloudinary.config({
+  cloudinary_name: process.env.CLOUDINARY_URL,
+});
+
+const storage = cloudinaryStorage({
+  cloudinary,
+  folder: 'forsetti',
+  allowedFormats: ['jpg', 'jpeg', 'png'],
+  transformation: [{
+    width: 500, height: 500, crop: 'limit',
+  }],
+});
+
+
+const upload = multer({
+  storage,
+  limits: {
+    fileSize: 2000000,
```

Comment:
Good job @BukkyOmo, I will like to verify what size exactly this is. I will like to suggest if you can make the maximum file size to to be 1mb. The reason for this is to reduce server load and upload time.